tools/validators/ontology_validator/yamlformat/tests/unit_lib_test.py

```diff
@@ -102,6 +102,18 @@ def testUnitFolderAddFromConfig(self):
     self.assertEqual(units['degrees_celsius'],
                      unit_lib.Unit('degrees_celsius', 'temperature', False))
 
+  def testUnitFolderAddFromConfigMultipleNoUnits(self):
+    doc = {
+      'powerfactor': [{'no_units': 'STANDARD'}],
```

Comment:
same here test the case where a measurement has two units: no_units, another_one